### PR TITLE
Create gitpod.md

### PR DESCRIPTION
**Pull request by BenLloydPearson**
#### Summary
This extracts the new GitPod deployment instructions introduced in #1170 so we can discuss them separately.

#### Ticket Link
n/a

### CODE REVIEW

File: site/content/contribute/more-info/getting-started/using-gitpod.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+#### :cyclone: Spinning up an environment
+
+1. Create a new workspace for a ticket/issue that you've claimed by going to the {{<newtabref href="https://github.com/mattermost/mattermost-gitpod-config/tree/master" title="mattermost-gitpod-config">}} repository and clicking the "Open in Gitpod" badge.
+
+    * You can also use the Gitpod extension to open up a repository in Gitpod, or manually prefix GitHub repository URLs with `gitpod.io/#`. What the extension does is add a green Gitpod button to repository pages on GitHub, and clicking it spins up a new environment for the repository on Gitpod.
```
Comment (mickmister) [inline site/content/contribute/more-info/getting-started/using-gitpod.md:17]:
```suggestion
    * You can also use the [Gitpod browser extension](https://www.gitpod.io/docs/configure/user-settings/browser-extension) to open up a repository in Gitpod, or manually prefix GitHub repository URLs with `gitpod.io/#`. What the extension does is add a green Gitpod button to repository pages on GitHub, and clicking it spins up a new environment for the repository on Gitpod.
```

**Review by mickmister — APPROVED** [review]
LGTM :+1: Just one suggested improvement to link to the Gitpod browser extension

**Comment by cwarnermm** [comment]
Emoji support needs to be enabled via Hugo to display the inline emojis present in this PR.
image

https://github.com/mattermost/mattermost-developer-documentation/pull/1170/files has emoji support enabled, where this content was originally developed.

@mickmister - Are you open to updating this PR to include emoji support? I believe that's all that's blocking this PR from being merged.

**Comment by mickmister** [comment]
@cwarnermm Should be fixed now :+1:

**Review by cwarnermm — APPROVED** [review]
Thanks so much, @mickmister, for getting this PR to the finish line!

**Pull request merged**